pages/index.js

```diff
@@ -1,7 +1,106 @@
-export default function HomePage() {
-	return (
-		<>
-			<h1>My Cosmetics Store</h1>
-		</>
-	);
+import styled from "styled-components";
+
+import { Btn } from "@/components/Button";
```

Comment:
- Using full names for components helps maintain consistency across the codebase, making it easier for new developers to understand the code.
- Following best practices in naming conventions helps in avoiding confusion and potential errors.
- This change should be adapt in the JSX as well

```suggestion
import { Button } from "@/components/Button";
```